src/amo/components/Addon/index.js

```diff
@@ -154,43 +160,31 @@ export class AddonBase extends React.Component {
           ? previewHeader.image_url
           : null;
 
-      const label = addon
-        ? i18n.sprintf(i18n.gettext('Preview of %(title)s'), {
-            title: addon.name,
-          })
-        : '';
-
-      const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
-
-      if (!previewURL && type === ADDON_TYPE_THEME) {
+      if (!previewURL && addon.type === ADDON_TYPE_THEME) {
         previewURL = addon.previewURL;
       }
 
-      const headerImage = (
-        <img
-          alt={label}
-          className="Addon-theme-header-image"
-          src={previewURL}
-        />
-      );
+      const themeClassName = 'Addon-theme-header';
 
       return (
-        <div
-          className="Addon-theme-header"
-          id="Addon-theme-header"
-          role="presentation"
-        >
-          {headerImage}
+        <div className={themeClassName} id={themeClassName} role="presentation">
+          <img
+            alt={label}
+            className="Addon-theme-header-image"
+            src={previewURL}
+          />
         </div>
       );
     }
 
-    const iconUrl = getAddonIconUrl(addon);
-
     return (
-      <div className="Addon-icon">
-        <img className="Addon-icon-image" alt="" src={iconUrl} />
-      </div>
+      <span className="Addon-icon">
```

Comment:
Ah well, that's a bit annoying to have to do that but I understand. Would adding `key` work though? I'd like to use `key` over `span` because it is more the react way.

Reply:
yep seems to works too so I can switch : )